spec/services/container_topology_service_spec.rb

```diff
@@ -11,8 +11,8 @@ def icon(f)
 
   describe "#build_kinds" do
     it "creates the expected number of entity types" do
-      allow(container_topology_service).to receive(:retrieve_providers).and_return([FactoryGirl.create(:ems_kubernetes)])
-      expect(container_topology_service.build_kinds.keys).to match_array([:Container, :Host, :Kubernetes, :Node, :Pod, :Replicator, :Route, :Service, :VM])
+      container_topology_service.stub(:retrieve_providers).and_return([FactoryGirl.create(:ems_kubernetes)])
```

Comment:
Any reason for this change? You're replacing the first line with a construct that's identical but obsolete.. (`.stub` is obsolete, `allow(...).to receive` is the Right Way)